aws_lab_setup/roles/manage_ec2_instances/defaults/main.yml

```diff
@@ -17,9 +17,8 @@ ec2_lab_node_types:
   - name: node2
     type: rhel7
 
-  - name: node3
-    type: rhel7
-
-  - name: haproxy
-    type: rhel7
+#  - name: node3
+#    type: rhel7
 
+#  - name: haproxy
+#    type: rhel7
```

Comment:
Rather than change the default topology, override it with an `extra_vars.yml` file. I don't believe this should be committed to the master branch.